phac-nml/irida-next
# Data Table: Projects Automated Pipelines

## What does this PR do and why?
This PR migrates the automated pipelines table to use the `data_table_component`

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/ebfb3d45-bf3e-4e61-98a4-45c03b349cc9)

## How to set up and validate locally
1. Verify the table updates accordingly after the following actions: Create, Edit, and Delete an updated pipeline
2. Create a pipeline with a very long name. Verify the 'left' column (id) is `sticky` during horizontal scrolling.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
